stock_picking_invoice_link/tests/test_stock_picking_invoice_link.py

```diff
@@ -272,7 +272,7 @@ def test_invoice_refund(self):
         )
         wiz_invoice_refund.modify_moves()
         new_invoice = self.so.invoice_ids.filtered(
-            lambda i: i.move_type == "out_refund" and i.state == "draft"
+            lambda i: i.move_type == "out_refund"
```

Comment:
This change doesn't seem related.

Reply:
The refund move should be posted, however not passing the is_modify parameter was causing it to stay in draft state,